packages/app/components/swipe-list.tsx

```diff
@@ -428,8 +430,9 @@ export const FeedItem = memo(
 
           <Reanimated.View
             style={[
-              tw.style("z-1 absolute bottom-0 right-0 left-0"),
```

Comment:
Here, `bottom-[${}]` was getting an absurd number event the parameter is not its. Needed to pass it as inline style.